src/pages/projects/build-modern-layouts-with-css-grid/index.tsx

```diff
@@ -0,0 +1,253 @@
+import React, {FunctionComponent} from 'react'
+import {NextSeo} from 'next-seo'
+import Markdown from 'react-markdown'
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
+import Image from 'next/image'
+import {find} from 'lodash'
+
+type LandingProps = {
+  course: any
+}
+
+const landingPage: FunctionComponent<LandingProps> = (props) => {
+  const {course} = props
+
+  const {
+    pricingPageFigmaUrl,
+    crmPageFigmaUrl,
+    pricingPageImageUrl,
+    crmPageImageUrl,
+    ogImage,
+  } = course?.projects
+
+  const introduction: any = find(course?.projects?.content, {
+    label: 'Introduction',
+  })
+  console.log('introduction', {introduction})
+  const challenges: any = find(course?.projects?.content, {label: 'Challenges'})
+  const pricingPageDescription: any = find(course?.projects?.content, {
+    label: 'Pricing Page description',
+  })
+  const crmPageDescription: any = find(course?.projects?.content, {
+    label: 'CRM page description',
+  })
+  const outcomes: any = find(course?.projects?.content, {label: 'Outcomes'})
+  const challengeResource: any = find(course?.projects?.content, {
+    label: 'Resource',
+  })
+  const pricingPageChallenge: any = find(course?.projects?.content, {
+    label: 'Pricing page challenge',
+  })
+  const dashboardPageChallenge: any = find(course?.projects?.content, {
+    label: 'Dashboard page challenge',
+  })
+  const studyQuestions: any = find(course?.projects?.content, {
+    label: 'Self-Study Questions',
+  })
+  const bestPractices: any = find(course?.projects?.content, {
+    label: 'Best Practices',
+  })
+  const Submission: any = find(course?.projects?.content, {label: 'Submission'})
```

Comment:
Sanity returns an array of content so accessing that data through lodash `find` so we don't have to worry about it in the view